app/assets/js/rtm-upload-terms.js

```diff
@@ -13,6 +13,18 @@ if ( 'object' === typeof rtMediaHook ) {
 
 		var terms_conditions_checkbox = jQuery( '#rtmedia_upload_terms_conditions' );
 
+		/**
+         * Show the warning message if the terms checkbox is enabled in the admin panel.
+         * Extra check if the HTML not exists, or has been removed intentionally.
+         *
+         * @author Adarsh Verma <adarsh.verma@rtcamp.com>
+         */
+        var activity_terms_enabled = rtmedia_upload_terms_data.activity_terms_enabled;
+        if( 'true' === activity_terms_enabled && 0 === terms_conditions_checkbox.length ) {
```

Comment:
There is an existing check which verifies existence of the element, is that required now since it is already checked here?

https://github.com/rtMediaWP/rtMedia/blob/8d2695d34a14e6127555f2e3a9dca984f7e311f9/app/assets/js/rtm-upload-terms.js#L16-L21

Could we instead update the existing code to return early if `activity_terms_enabled` is not enabled, if it is enabled, then check element existence to handle cases in `if/else`?

Reply:
Thanks for this suggestion @thrijith. The latest commit contains the code changes.